cmake.md

```diff
@@ -0,0 +1,30 @@
+## CMake build system
+### Terminology
+
+The CMake system in symbiflow-arch-defs uses the following hierarchy:
+
+* ARCH - The architecture of FPGA.  This is used to capture architecture level parameters (e.g. tech libraries).
+* DEVICE_TYPE - A type of device.  This is defined as an ARCH and the arch.xml which defines the units with this device type.
+* DEVICE - A specific instance of a DEVICE_TYPE.  A device definition is a DEVICE_TYPE, a PACKAGE list, and IO map definitions for each package if needed.
+* BOARD - A specific DEVICE and PACKAGE instance, along with a PROG_TOOL/PROG_CMD.  A board is what is targeted by ADD_FPGA_TARGET.
+
+### Creating a new FPGA target
+
+Assuming that the board(s) you are interested in targetting is already defined, ADD_FPGA_TARGET is what will create targets.  See [ADD_FPGA_TARGET documentation](https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/make/devices.cmake#L559).
```

Comment:
FYI You should be able to use a relative link here. IE
`[ADD_FPGA_TARGET documentation](../make/devices.cmake#L559)`

Reply:
Fixed